### PR TITLE
Testing comment feature with very minor update

### PR DESCRIPTION
**Pull request by wadepickett**
Testing the line comment feature with a tiny grammar fix.

### CODE REVIEW

File: aspnetcore/razor-pages/razor-pages-conventions.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ Use <xref:Microsoft.AspNetCore.Mvc.RazorPages.RazorPagesOptions.Conventions> to
 
 ## Page route action conventions
 
-The default route model provider that derives from <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageRouteModelProvider> invokes conventions which are designed to provide extensibility points for configuring page routes.
+The default route model provider that derives from <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageRouteModelProvider> invokes conventions that are designed to provide extensibility points for configuring page routes.
```
Comment (wadepickett) [inline aspnetcore/razor-pages/razor-pages-conventions.md]:
```suggestion
The default route model provider that derives from <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageRouteModelProvider> invokes conventions that are designed to provide extensibility points for configuring page routes.
<!-- this is a comment -->
```

**Comment by wadepickett** [comment]
line commenting works fine.  Thanks @Rick-Anderson.   Closing this PR.  Sorry, I should have done a draft instead, wasn't thinking.

**Comment by Rick-Anderson** [comment]
It's not line commenting, it's add suggestion to batch.

**Comment by wadepickett** [comment]
Commit suggestion works for me now, thanks @Rick-Anderson

**Comment by Rick-Anderson** [comment]
@wadepickett  can you show me that my suggestions are available for you to commit?

File: aspnetcore/razor-pages/razor-pages-conventions.md [hunk 2]
```diff
@@ -392,7 +393,7 @@ Visit the Contact page at either its ordinary route, `/Contact`, or the custom r
 
 ## Page model action conventions
 
-The default page model provider that implements <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageApplicationModelProvider> invokes conventions which are designed to provide extensibility points for configuring page models. These conventions are useful when building and modifying page discovery and processing scenarios.
+The default page model provider that implements <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageApplicationModelProvider> invokes conventions that are designed to provide extensibility points for configuring page models. These conventions are useful when building and modifying page discovery and processing scenarios.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/razor-pages-conventions.md]: The default page model **provider** that implements <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageApplicationModelProvider> invokes conventions that are designed to provide extensibility points for configuring page models. These conventions are useful when building and modifying page discovery and processing scenarios.

**Comment by Rick-Anderson** [comment]
IT's not working. I can post this to the GitHub issue.

File: aspnetcore/razor-pages/razor-pages-conventions.md [hunk 3]
```diff
@@ -392,7 +393,7 @@ Visit the Contact page at either its ordinary route, `/Contact`, or the custom r
...
 
 For the examples in this section, the sample app uses an `AddHeaderAttribute` class, which is a <xref:Microsoft.AspNetCore.Mvc.Filters.ResultFilterAttribute>, that applies a response header:
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/razor-pages-conventions.md]: For examples in this section, the sample app uses an `AddHeaderAttribute` class, which is a <xref:Microsoft.AspNetCore.Mvc.Filters.ResultFilterAttribute>, that applies a response header:

**Comment by wadepickett** [comment]
@Rick-Anderson: Thanks, I sent you screen shots of what I am seeing on my end as well.

File: aspnetcore/razor-pages/razor-pages-conventions.md (hunk 3 above)
Comment (wadepickett) [reply aspnetcore/razor-pages/razor-pages-conventions.md]: No "Commit Suggestion" option available to me.  Thanks for the test.

**Comment by Rick-Anderson** [comment]
@wadepickett  let's close this and create a new clean PR

**Comment by wadepickett** [comment]
Submitted new clean pr for test here:  #16446

**Pull request closed without merging**